inst/metadata/calibration_run_options.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
I feel we use spaces elsewhere for indentation - might be worth updating this so we don't end up with a weird mix of tabs and spaces

Reply:
Thanks, hadn't noticed that but I've switched all tabs in options to 2 spaces.

Reply:
I think the main culprit here is my Emacs config defaulting to tabs and sneaking them in there!! I need to figure that out.